.github/workflows/check_tflite_files.yml

```diff
@@ -19,10 +19,11 @@ jobs:
         run: |
           URL="https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files"
           PR_FILES=$(curl -s -X GET -H "Authorization: Bearer ${{ secrets.TFLM_BOT_REPO_TOKEN }}" $URL | jq -r '.[] | .filename')
-          rm -rf /tmp/pull_request_files.txt
-          echo "${PR_FILES}" >> /tmp/pull_request_files.txt
-
-          python3 ci/check_tflite_files.py /tmp/pull_request_files.txt
+          rm -rf tmp_pull_request_files.txt
+          echo "${PR_FILES}" >> tmp_pull_request_files.txt
+          rm -rf .git
+          echo ${{ secrets.TFLM_BOT_PACKAGE_READ_TOKEN }} | docker login ghcr.io -u tflm-bot --password-stdin
+          docker run --rm -v `pwd`:/working -w /working ghcr.io/tflm-bot/tflm-ci:latest python3 ci/check_tflite_files.py tmp_pull_request_files.txt
```

Comment:
can we change the name of the directory from working to something else? like /tflite-micro?